src/js/modules/authentication/actions.js

```diff
@@ -42,7 +42,13 @@ function loginFail(err) {
  * @return {Object} - the action
  */
 function loginSuccess(logindata) {
-    sessionStorage.setItem('account_jwt', logindata.jwt);
+
+    try {
+        sessionStorage.setItem('account_jwt', logindata.jwt);
+    }
+    catch (e) {
```

Comment:
a `console.error()` call can probably go in the catch block to indicate when an error actually happens here, that way if something unexpected happens the error won't be hidden from us.